plugin.audio.soundcloud/addon.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.soundcloud" name="SoundCloud" version="3.0.0" provider-name="jaylinski">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.future" version="0.16.0"/>
+        <import addon="script.module.requests" version="2.4.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="de_DE">Soundcloud ist eine Online-Audio-Vertriebsplattform.</summary>
+		<summary lang="en_GB">SoundCloud is an online audio distribution platform.</summary>
+        <language></language>
+        <platform>all</platform>
+        <license>MIT</license>
+        <forum>https://forum.kodi.tv/showthread.php?tid=206635</forum>
+        <website>https://soundcloud.com</website>
+        <source>https://github.com/jaylinski/kodi-addon-soundcloud</source>
+        <news>3.0.0 (2019-0x-xx)
+[new] complete rewrite
```

Comment:
news tag is only for the latest version, if you need to keep a changelog along with the code use the changelog.txt (although deprecated)

Reply:
Fixed. I also replaced the temporary date with a real date.